c8/client.py

```diff
@@ -26,9 +26,9 @@ class C8Client(object):
 
     def __init__(
         self,
-        protocol="http",
+        protocol="https",
```

Comment:
Hello @the-jesus-aviles thanks for this! We are glad you are working with our sdk! We would encourage you to keep working with the sdk and let us know whenever you face any issue!

Reply:
Also, we would encourage you to join our community slack as well:
https://join.slack.com/t/macrometa/shared_invite/zt-1idwblep2-14Vskc8nrIDxTAk0RAZPEw

Reply:
👍